crownstone_uart/Exceptions.py

```diff
@@ -15,18 +15,5 @@ def __str__(self) -> str:
         return self.value
 
 
-class UartManagerError(Enum):
-    """Error types for Uart Manager exceptions."""
-    UART_BRIDGE_ERROR                   = "UART_BRIDGE_ERROR"
-
-    def __str__(self) -> str:
-        """Return value of the error."""
-        return self.value
-
-
-class UartBridgeException(Exception):
-    """Raised on errors in the Uart Bridge."""
-    
-
-class UartManagerException(Exception):
-    """Raised on errors in the Uart Manager."""
\ No newline at end of file
+class UartException(Exception):
+    """Raised on errors in the Uart Manager."""
```

Comment:
Isn't removing the old exceptions is a breaking change?

Reply:
I've added those in the previous PR, which wasn't released yet